src/DataSheet.js

```diff
@@ -67,6 +67,7 @@ export default class DataSheet extends PureComponent {
       forceEdit: false,
       editing: {},
       clear: {},
+      pressedKey: null,
```

Comment:
Where is this actually being used? I see it being read from `this.props` in `renderRow` but I don't see how that relates to this.